proto/storage/alert.proto

```diff
@@ -237,7 +238,8 @@ message ListAlertDeployment {
 
 enum ViolationState {
   ACTIVE = 0;
-  SNOOZED = 1;
+  reserved 1;
```

Comment:
Will this need to change any validations anywhere else in the code?

Reply:
According to the power of grep, I think I got them.  I don't think it was referenced in many places really since it has been deprecated for 5 years.